lib/core/seleniumRunner.js

```diff
@@ -484,19 +515,42 @@ class SeleniumRunner {
   async runScriptInCategory(category, isAsync) {
     const scriptNames = Object.keys(category);
     const results = {};
+    let requires = [];
+
     for (let scriptName of scriptNames) {
-      try {
-        const script = category[scriptName];
-        const result = await this.runScriptFromCategory(
-          script,
-          isAsync,
-          scriptName
-        );
-        if (!(result === null || result === undefined)) {
-          results[scriptName] = result;
+      let script = category[scriptName].content;
+      //
```

Comment:
nit: Drop the `//\n` lines (throughout).  The formatting (from ESlint?) is already expanding vertical spacing by a great deal.

Reply:
Just carryover of personal style, that's all. I can blame eslint for many ills -- this is not one of them! :-)